portalnet/src/config.rs

```diff
@@ -33,7 +35,7 @@ impl Default for PortalnetConfig {
             external_addr: None,
             private_key: B256::random(),
             listen_port: 4242,
-            bootnodes: Bootnodes::default(),
+            bootnodes: Bootnodes::default().to_enrs(Network::Mainnet),
```

Comment:
nit: I don't like that `PortalnetConfig` implements `Default` that shouldn't be used in production code (there is a chance it can be used by mistake).
Currently it's used only in tests (as far as I can tell), so I would remove this and add something like this:

```rust
impl PortalnetConfig {
    pub fn new(trin_config: &TrinConfig, private_key: B256) -> Self {
        // current function
    }

    #[cfg(test)]
    pub fn new_for_tests() -> Self {
        Self {
            // current default
        }
    }
}
```

But, it isn't really a big deal and not really part of this PR, so feel free to ignore it (more of an idea of how we can do this better in general).
Up to you.

Reply:
Well, we do use the `Default` trait quite a bit...
- inside `PortalnetConfig::new()`
- in "test" utils that which don't seem to be scoped to the test compilation set (https://github.com/ethereum/trin/blob/master/utp-testing/src/lib.rs#L164)
- inside our unit tests 

Ultimately, the `Default` syntax is quite handy for initializing a new instance with only a few adjusted fields (https://github.com/ethereum/trin/blob/master/portalnet/src/overlay/service.rs#L2779)...

I get your concern about using it inside "production" code, so I removed it from the `PortalnetConfig::new()` constructor. 
- the only fields that were being set by the `Default` trait were...
  - `internal_ip` - which we actually never use, so I removed it
  - `node_addr_cache_capacity` - which I just updated to use the default value. Maybe this should be added as a cli arg?

And I added a comment to the `Default` trait that it should only be used inside testing (obvs. this won't be caught by the compiler)